src/php/bin/generate_proto_php.sh

```diff
@@ -39,10 +39,12 @@ protoc --proto_path=src/proto/math \
 
 # replace the Empty message with EmptyMessage
 # because Empty is a PHP reserved word
-sed -i 's/message Empty/message EmptyMessage/g' \
-    src/proto/grpc/testing/empty.proto
-sed -i 's/grpc\.testing\.Empty/grpc\.testing\.EmptyMessage/g' \
-    src/proto/grpc/testing/test.proto
+sed 's/message Empty/message EmptyMessage/g' \
+  src/proto/grpc/testing/empty.proto > empty.proto
```

Comment:
I did a `cd $(dirname $0)/../../..` earlier in the script. That seems to be good enough. I tested this on linux and mac and both seem to be working